portal/src/main/webapp/js/api/cbioportal-datamanager.js

```diff
@@ -1049,7 +1049,8 @@ window.initDatamanager = function (genetic_profile_ids, oql_query, geneset_ids,
      * @property {string} study - identifier of the study to which this sample belongs
      * @property {string} sample - identifier of the sample
      * @property {number} uid - cross-study identifier of this sample in the current Oncoprint
-     * @property {number} profile_data - score of the sample or patient for this gene set
+     * @property {?number} profile_data - score of the sample for this gene set
```

Comment:
A prefixed question mark is the JSDoc syntactic sugar for ‘nullable’: http://usejsdoc.org/tags-type.html

Since we don't actually use JSDoc to generate API docs, and we had the plan to eventually express this in TypeScript, I think it would be useful if I rephrase this to an explicit union type `number|null` to avoid this confusion while thinking in TypeScript.